example/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "pixelpact-example",
+  "version": "1.0.0",
+  "description": "An example usage of pixelpact for a static HTML page",
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "start": "node index.js",
+    "format": "npx prettier --write ."
+  },
+  "dependencies": {
+    "node-fetch": "^3.2.10"
```

Comment:
Ah, yes, since [Node 18](https://nodejs.org/en/blog/announcements/v18-release-announce#fetch-experimental) it's enabled (but still experimental). Will adapt it :+

Reply:
Regarding Deno: I think the main Problem is that Playwright does not officially support Deno